os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -97,6 +99,18 @@ void mm_manage_alloc_fail(struct mm_heap_s *heap, int startidx, int endidx, size
 {
 	irqstate_t flags = enter_critical_section();
 
+	/* If secure state, do not print memory usage and address infomation */
```

Comment:
Don't you need to move this after cpu pause (line 119)?

Reply:
hmmm... little bit duplicated and missing conditional.
How about below?
```
void mm_manage_alloc_fail(...)
{
    enter_critical_section();
    cpu pause all
    abort_mode = true;
    write reboot reason ();

    if (!IS_SECURE_STATE()) {   <---- only if is added, rest are existed codes.
       heap messages
    }

   cpu resume all
   panic();
   leave_critical_section();
}
```

Reply:
we don't need to call cpu pause. 
the cpu pause is for only log mixing

Reply:
I removed duplicated code.